docs/client-libraries-tableviews.md

```diff
@@ -4,36 +4,105 @@ title: Work with TableView
 sidebar_label: "Work with TableView"
 ---
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+````
+
 After setting up your clients, you can explore more to start working with [TableView](concepts-clients.md#tableview).
 
 ## Configure TableView
 
-The following is an example of how to configure a TableView.
 
-```java
-TableView<String> tv = client.newTableViewBuilder(Schema.STRING)
-  .topic("my-tableview")
-  .create()
-```
+````mdx-code-block
+<Tabs groupId="lang-choice"
+defaultValue="Java"
+values={[{"label":"Java","value":"Java"},{"label":"C++","value":"C++"}]}>
+<TabItem value="Java">
+
+  The following is an example of how to configure a TableView.
+  ```java
+    TableView<String> tv = client.newTableViewBuilder(Schema.STRING)
+    .topic("my-tableview")
+    .create()
+  ```
+
+  You can use the available parameters in the `loadConf` configuration or related [API](/api/client/2.10.0-SNAPSHOT/org/apache/pulsar/client/api/TableViewBuilder.html) to customize your TableView.
```

Comment:
Suggest 
1) using a variable instead of a fixed version 
2) showing the API name explictly

```suggestion
You can use the available parameters in the `loadConf` configuration or the API [`TableViewBuilder`](/api/client/@pulsar:version_number@/org/apache/pulsar/client/api/TableViewBuilder.html) to customize your TableView.
```

@tisonkun double check: is it correct to use this variable (`@pulsar:version_number@`)? Thanks!